.internal-ci/helm/fog-services/values.yaml

```diff
@@ -138,8 +186,40 @@ fogView:
     value: 'true'
     effect: NoSchedule
 
+fogLedgerRouter:
+  replicaCount: 3
+  podManagementPolicy: Parallel
+
+  podAnnotations:
+    fluentbit.io/include: 'true' # collect logs with fluentbit
+    fluentbit.io/exclude-jaeger-agent: 'true'
+
+  image:
+    org: ''
+    name: fog-ledger
+    pullPolicy: Always
+
+  rust:
+    backtrace: full
+    log: info,rustls=warn,hyper=warn,tokio_reactor=warn,mio=warn,want=warn,reqwest=warn,rusoto_core=error,rusoto_signature=error,h2=error,rocket=warn,<unknown>=warn
+
+  resources:
+    limits:
+      intel.com/sgx: 5000
+    requests:
+      intel.com/sgx: 5000
+
+  nodeSelector:
+    sgx-enabled-node: 'true'
+
+  tolerations:
+  - key: sgx
+    operator: Equal
+    value: 'true'
+    effect: NoSchedule
+
 fogLedger:
-  replicaCount: 2
+  replicaCount: 3
```

Comment:
Is this now the router or the store replica count

Reply:
Will address in follow-up PR.  Needs spelunking.